src/tests/ftest/osa/osa_online_extend.py

```diff
@@ -150,16 +88,15 @@ def run_online_extend_test(self, num_pool):
         # Create a pool
         pool = {}
         pool_uuid = []
-        total_servers = len(self.hostlist_servers)
 
-        # Extend one of the ranks (or server)
-        # rank index starts from zero
-        rank = total_servers
+        # Extend one of the ranks 4 and 5
+        rank = [4, 5]
 
         # Start the daos_racer thread
-        daos_racer_thread = threading.Thread(target=self.daos_racer_thread)
-        daos_racer_thread.start()
-        time.sleep(30)
+        if racer is True:
+            daos_racer_thread = threading.Thread(target=self.daos_racer_thread)
+            daos_racer_thread.start()
+            time.sleep(30)
```

Comment:
This is a big sleep. What is the intention of sleep here? Are we waiting for daos_racer to start or finish?
If waiting it to finish we can use daos_racer_thread.join()

Reply:
daos_racer runs in parallel during the whole OSA test execution... daos_racer creates pools and starts sending IO on a separate pool... While daos_racer initial setup is happening, creating a TestPool causes pool creation failures... That is the reason for the large timeout... Di is not fond of daos_racer tool running during OSA testing (due to bad IO data sent by the tool). To isolate the rebuild issues under OSA, we are temporarily stubbing out daos_racer tool.